adapter/internal/oasparser/model/types.go

```diff
@@ -265,6 +296,19 @@ func (apiProject *ProjectAPI) ProcessFilesInsideProject(fileContent []byte, file
 		}
 		apiProject.APIYaml = apiYaml
 		ExtractAPIInformation(apiProject, apiYaml)
+	} else if strings.Contains(fileName, rateLimitPoliciesFile) {
+		loggers.LoggerAPI.Debugf("fileName : %v available for the API project.", fileName)
+		rlPoliciesJsn, conversionErr := utills.ToJSON(fileContent)
+		if conversionErr != nil {
+			loggers.LoggerAPI.Errorf("Error occured rate limit policies file to json: %v", conversionErr.Error())
```

Comment:
Shall we follow a uniform way to print the error? We can either print the error after a `:` or `.`.

Reply:
The pattern used in https://github.com/wso2/product-microgateway/pull/3142/files#diff-b080d837ff8a66b69ebee0933209fe91291049eb25a265a5afb5d78220be3bb8R64 also is clear.